test/org/loklak/harvester/TwitterScraperTest.java

```diff
@@ -43,8 +39,8 @@ public void test_prepareSearchURL() {
     }
 
     @Test
-    public void test_search() {
```

Comment:
I added 2 methods in TwitterScraper to just create a test it to initialize automated tests. More tests are in pipeline (working on them) when this pr gets merged.